Failed to build xen-tools

**Description**
Failed to build xen-tools

**Steps to reproduce the issue:**
1. Put the dev machine behind the firewall
2. export https_proxy and http_proxy
3. run:
build-tools/bin/linuxkit pkg build pkg/xen-tools

**Describe the results you received:**
make[4]: Leaving directory '/xen/tools/xcutils'
make[3]: Leaving directory '/xen/tools'
make[3]: Entering directory '/xen/tools'
make -C firmware install
make[4]: Entering directory '/xen/tools/firmware'
GIT=git /xen/tools/firmware/../../scripts/git-checkout.sh git://xenbits.xen.org/seabios.git rel-1.12.0 seabios-dir
Cloning into 'seabios-dir-remote.tmp'...
fatal: unable to connect to xenbits.xen.org:
xenbits.xen.org[0: 104.239.192.120]: errno=Operation timed out
make[4]: *** [Makefile:31: seabios-dir] Error 128
make[4]: Leaving directory '/xen/tools/firmware'
make[3]: *** [/xen/tools/../tools/Rules.mk:251: subdir-install-firmware] Error 2
make[3]: Leaving directory '/xen/tools'
make[2]: Leaving directory '/xen/tools'
make[2]: *** [/xen/tools/../tools/Rules.mk:246: subdirs-install] Error 2
make[1]: *** [Makefile:74: install] Error 2
make[1]: Leaving directory '/xen/tools'
make: *** [Makefile:127: install-tools] Error 2
The command '/bin/sh -c make && make dist' returned a non-zero code: 2
exit status 2


**Describe the results you expected:**
the alpine docker should be able to do:
git://xenbits.xen.org/seabios.git

**Additional information you deem important (e.g. issue happens only occasionally):**

--- a/pkg/xen-tools/Dockerfile
+++ b/pkg/xen-tools/Dockerfile
@@ -1,4 +1,5 @@
 FROM alpine:3.8 as kernel-build
+ENV GIT_HTTP=y
 # hadolint ignore=DL3018
 RUN apk add --no-cache \
     gcc make libc-dev dev86 xz-dev perl bash python-dev \

